filebeat/docs/getting-started.asciidoc

```diff
@@ -56,48 +56,14 @@ include::{libbeat-dir}/tab-widgets/install-widget.asciidoc[]
 include::{libbeat-dir}/shared/connecting-to-es.asciidoc[]
 
 [float]
-[[enable-modules]]
-=== Step 3: Enable and configure data collection modules
+[[collect-data]]
+=== Step 3: Collect data
 
 {beatname_uc} uses modules to collect and parse log data.
+You can also use {beatname_uc} to collect application logs.
+Select your use case below.
```

Comment:
I don't think the tabbed layout works. The tabbed layout makes me think it's a choice between two different, mutually exclusive ways of collecting log data. It also feels like the tab labels aren't parallel. One is the thing you use to do the monitoring, the other is the thing being monitored.

Reply:
I also think it's possible that users might be configuring modules for some services *and* inputs for their application logs. In that case, they aren't mutually exclusive ways to collect data, so the tabbed layout would make less sense.

Reply:
These are both good points. I'll likely update this PR to include an additional quick start. I am worried about how we'll avoid content duplication with another quick start though. Tagged regions?

Reply:
Agh, but the more I think about it, the more I think it makes sense to roll with Felix's original idea of having both modules and application logs in one quick start.

Reply:
This is really frying my brain